scripts/import_suppliers.py

```diff
@@ -0,0 +1,322 @@
+          ],
+          "dunsNumber": "733053339",
+          "eSourcingId": null,
+          "clients":
+          [
+            "Home Office",
+            "Metropolitan Police",
+            "Aviva",
+            "ITV",
+          ]
+        }
+        """
+        # variable either set to empty string or value of `clientsString` key
+        clients_string = '' if 'clientsString' not in json_from_file.keys() \
+            else json_from_file['clientsString']
+
+        # `clientsString` list of (comma-separated) clients or empty
+        json_from_file['clientsString'] = \
+            SupplierPutter.split_comma_separated_value_string_into_list(
+                clients_string
+            )
+
+        json_from_file = SupplierPutter.change_key_names(
+            json_from_file,
+            [
+                ['clientsString', 'clients'],
+                ['contactEmail', 'email'],
+                ['contactPhone', 'phoneNumber'],
+                ['supplierId', 'id']
+            ]
+        )
+
+        # key/values nested behind `contactInformation` object
+        json_from_file = SupplierPutter.nest_key_value_pairs(
+            json_from_file,
+            'contactInformation',
+            ['email', 'website', 'phoneNumber', 'contactName', 'address']
+        )
+
+        # key/values in `address` un-nested (flush with `contactInformation`)
+        json_from_file['contactInformation'] = \
+            SupplierPutter.un_nest_key_value_pairs(
+                json_from_file['contactInformation'],
+                'address'
+            )
+
+        json_from_file['contactInformation'] = \
+            SupplierPutter.convert_a_json_object_into_an_array_with_one_entry(
+                json_from_file['contactInformation']
+            )
+
+        json_from_file = \
+            SupplierPutter.convert_values_to_utf8_except_blacklisted_keys(
+                json_from_file,
+                ['id', 'clients', 'contactInformation']
+            )
+
+        return json_from_file
+
+    @staticmethod
+    def convert_values_to_utf8_except_blacklisted_keys(obj, blacklist):
+        for key in obj.keys():
+            if key not in blacklist:
+                obj[key] = SupplierPutter.convert_string_to_utf8(obj[key])
+
+        return obj
+
+    @staticmethod
+    def convert_string_to_utf8(string):
+        # check if it's a string value
+        if isinstance(string, str):
+            return unicode(string, 'utf-8')
+
+        else:
+            return string
+
+    @staticmethod
+    def split_comma_separated_value_string_into_list(string):
+        if not string:
+            return []
+
+        return [value for value in
+                map(unicode.strip, string.split(',')) if len(value) is not 0]
+
+    @staticmethod
+    def un_nest_key_value_pairs(obj, key_of_nested_obj):
+        if key_of_nested_obj not in obj.keys():
+            # abort(400, "No '{0}' key found".format(key_of_nested_obj))
+            # TODO: FAIL LOUDLY
```

Comment:
Are the TODOs in this script going to be done, or are they unnecessary now?  If the former, then they should be done now as this is a one-off script and there will be no point doing them later.   If the latter then please delete the comments.